packages/explorer-2.0/lib/utils.tsx

```diff
@@ -45,7 +45,7 @@ export const getDelegatorStatus = (
   if (
     !delegator ||
     !delegator.bondedAmount ||
-    parseFloat(Utils.fromWei(delegator.bondedAmount)) === 0
+    parseFloat(delegator.bondedAmount) === 0
```

Comment:
This whole expression (L46-L48) can be shortened I think

```suggestion
    !+delegator?.bondedAmount
```

```javascript
let a = {x:0}

!+a?.x
// true
!+a?.y
// true
```